src/main/java/step5/domain/Car.java

```diff
@@ -0,0 +1,47 @@
+package step5.domain;
+
+public class Car {
+    private static final int INITIAL_POSITION = 1;
+    private static final int CONDITION = 4;
+    private static final int NAME_LENGTH = 5;
+    private String name;
```

Comment:
```suggestion
    private final String name;
```
이름의 경우 불변하게 만들어주면 좋을 것 같아요 😄